### PR TITLE
docs: create simple getting started

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Close: #961

### CODE REVIEW

**Review by danielbate — APPROVED** [review]

File: apps/docs/src/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Getting Started with Fuels-ts
+
+This guide will walk you through the process of setting up and using the Fuels-ts library in your front-end project.
+
+## Installing the Fuels-ts Library
+
+To begin, you need to add the `fuels` dependency to your project. You can do this using the following command:
+
+::: code-group
```
Comment (arboleya) [inline apps/docs/src/getting-started.md:9]: 🪄

File: apps/docs/src/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+```tsx
+import { BN, Provider, Wallet } from "fuels";
+import { useEffect, useState } from "react";
+
+function App() {
+  const [balance, setBalance] = useState(0);
+
+  const provider = new Provider("https://beta-3.fuel.network/graphql");
+  const myWallet = Wallet.fromAddress("wallet-address", provider);
```
Comment (arboleya) [inline apps/docs/src/getting-started.md:39]: I think the `wallet-address` value might go unnoticed.

```suggestion
  const myWallet = Wallet.fromAddress("0x...", provider);
```

**Review by arboleya — DISMISSED** [review]
LGTM. ✨ 

Left some optional comments/suggestions.

**Comment by Torres-ssf** [comment]
@FuelLabs/sdk-ts I've committed some suggestions now I need the approval again :pray:

**Review by arboleya — APPROVED** [review]

**Review by Dhaiwat10 — APPROVED** [review]

**Pull request merged**